ci/infra/testrunner/tests/test_node_reboot.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import json
+import time
+
+import pytest
+from timeout_decorator import timeout
+
+logger = logging.getLogger("testrunner")
+
+
+@timeout(120)
+def wait_for_node_reboot(node_type, node_number, platform):
+    while True:
+        try:
+            platform.ssh_run(node_type, node_number, 'echo hello')
+        except Exception as ex:
+            logger.debug(ex)
```

Comment:
We are logging here errors that are expected to happen as we are waiting for the node to reboot. That may not be necessary.